server/src/dtos/asset-response.dto.ts

```diff
@@ -43,7 +48,7 @@ export class AssetResponseDto extends SanitizedAssetResponseDto {
   exifInfo?: ExifResponseDto;
   smartInfo?: SmartInfoResponseDto;
   tags?: TagResponseDto[];
-  people?: PersonWithFacesResponseDto[];
+  people?: PeopleWithFacesResponseDto;
```

Comment:
This is a breaking change! I'm not really sure how to handle this case, as it isn't covered by the `PropertyLifecycle` idea. The proper approach would probably be to introduce a new variable? Interested for @jrasm91 opinion on this though.

Reply:
Yep that's a breaking change. It's fine to have another variable, but I prefer to have everything in `people`. 

Another simple fix would be to send all faces: currently we only filter faces associated with a person. But I prefer to represent faces/people like that:
```json
"people": {
    "faces": [
        {
        "birthDate": null,
        "id": "b70e6b20-2dce-4184-aa20-dac1c9717a16",
        "isHidden": false,
        "name": "Test Person",
        "thumbnailPath": "thumbnail.jpg"
	"faces": [
		{
			"id": "ef8c6a56-2891-424b-a521-11c673e9bfd3",
			"boundingBoxX1": 1494,
			"boundingBoxX2": 1705,
			"boundingBoxY1": 430,
			"boundingBoxY2": 733
		}
	]
        }
    ],
    "numberOfFaces": 2
}
```
{
	"0": {
		"id": "f85c7e91-148e-4fca-847a-6f9284745283",
		"imageHeight": 1440,
		"imageWidth": 1920,
		"boundingBoxX1": 734,
		"boundingBoxX2": 860,
		"boundingBoxY1": 643,
		"boundingBoxY2": 822
	}
}